distributed/tests/test_client.py

```diff
@@ -301,17 +301,17 @@ def test_gather_strict(c, s, a, b):
 
 @gen_cluster(client=True, timeout=None)
 def test_get(c, s, a, b):
-    result = yield c._get({'x': (inc, 1)}, 'x')
+    result = yield c.get({'x': (inc, 1)}, 'x', sync=False)
```

Comment:
We should probably have a test or two for the behavior of `get(sync=False)` producing normal futures.